protocol/requests/mute_chat_request.go

```diff
@@ -0,0 +1,21 @@
+package requests
+
+import (
+	"errors"
+)
+
+var ErrInvalidMuteChatParams = errors.New("mute-chat: invalid params")
+
+type MuteParams struct {
```

Comment:
The name of this struct should be `MuteChat`, we use the name of the endpoint for the request